imjoy/runners/subprocess.py

```diff
@@ -187,8 +191,8 @@ def logging_callback(msg, type="info"):  # pylint: disable=redefined-builtin
             )
             asyncio.run_coroutine_threadsafe(coro, eloop).result()
 
-        args = '{} "{}" --id="{}" --server={} --secret="{}"'.format(
-            cmd, TEMPLATE_SCRIPT, pid, "http://127.0.0.1:" + engine.opt.port, secret_key
+        args = '{} -m workers.python_worker --id="{}" --server={} --secret="{}"'.format(
```

Comment:
What if `cmd` isn't `python`?

Reply:
it will fail of course, we kind of assume cmd is a python command.
We can potentially check that, but it will be useless I guess, because one can add environment variable before the command.

Reply:
I think we don't need to consider that, we made it clear in docs about what should be used in `cmd`.